tests/ui/float_cmp.rs

```diff
@@ -77,6 +77,13 @@ fn main() {
 
     assert_eq!(a, b); // no errors
 
+    let a1: [f32; 1] = [0.0];
+    let a2: [f32; 1] = [1.1];
```

Comment:
Could you also add a test for the type `[&f32; 1]`?

Reply:
Thanks for the help.